base/applications/regedit/lang/de-DE.rc

```diff
@@ -441,7 +429,7 @@ STRINGTABLE
 BEGIN
     ID_EDIT_MODIFY "Ändert die Daten eines Wertes"
     ID_EDIT_NEW_KEY "Fügt einen neuen Schlüssel hinzu"
-    ID_EDIT_NEW_STRINGVALUE "Fügt eine neue Zeichenkette hinzu "
+    ID_EDIT_NEW_STRINGVALUE "Fügt eine neue Zeichenkette hinzu"
```

Comment:
Have you checked whether the other translations don't have this space?

Reply:
Yes, I checked them all. And no other language does have this unintended space.